### ISSUE TITLE
Accessibility feedback: the Publish button placement is confusing

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#3697]
Reporting some first feedback from real assistive technologies users, see https://make.wordpress.org/accessibility/2017/11/28/screen-reader-user-experience-with-gutenberg/

> Well I doodled with the editor, the best thing about it, well there isn’t, I guess the heading where the title should be is nice. The rest, I was able to add a block of text, typing it up **but the publish menu I couldn’t exactly navigate, I preferred the button as it was**.

As already mentioned a few times on other issues, the placement of the Publish button is not ideal, as users are forced to navigate tabbing backwards to find it.

In the Classic Editor, in terms of keyboard navigation, the publish button is placed _after_ the editor. This matches a logical flow, where users first add a title, add a content, and then they find the publish options and button.

In Gutenberg, the Publish button is placed _before_ the editor. As mentioned a few times before, this has nothing to do with the _visual_ placement. What matters here is the placement of the button in the markup, and thus its position in the tab sequence.

For keyboard and screen reader users, navigating content is a linearized experience. Placing the Publish button before the content is just counterintuitive and doesn't match a common sense logical flow where users first edit the content, and then they publish.

**Comment by afercia** [comment]
For some previous discussion about this issue, see #467 "Editor toolbar: controls order and grouping" then renamed to "Add keyboard shortcuts à la Slack to jump through the main UI sections"

**Comment by afercia** [comment]
Discussed during today's accessibility bug-scrub. Worth noting the Publish button and related panel have been greatly improved recently in #7552. Pending some more testing, this issue is a close-candidate.

**Comment by aaronjorbin** [comment]
I'm going to close based on the last comments. We can reopen if there are still major concerns.
